php-checks/src/test/resources/checks/StringLiteralDuplicatedCheck/default.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+echo "aa()";    // OK - too short
+echo "aa()";
+echo "aa()";
+echo "aa()";
+echo "aa()";
+
+echo "bbbbbb";  // OK - too few
```

Comment:
This comment is misleading: this is compliant, but for another reason. And I think we don't have a test for "too few".